jobs/JREGIONAL_RUN_POST

```diff
@@ -110,20 +110,6 @@ cd_vrfy "${fhr_dir}"
 #
 #-----------------------------------------------------------------------
 #
-# Make sure that fhr is a non-empty string consisting of only digits.  
```

Comment:
Instead of getting rid of this check here and in JREGIONAL_RUN_WGRIB2, should we modify them to allow for sub-hourly fhr specifications?

Reply:
This check is not true for sub-hours. It is good to have this check for sub-hours but I think the scripts have enough check to fhr.

Reply:
We can add them back later.